phpBB/includes/functions_display.php

```diff
@@ -1119,13 +1119,18 @@ function display_custom_bbcodes()
 		// Convert Numeric Character References to UTF-8 chars.
 		$row['bbcode_helpline'] = utf8_decode_ncr($row['bbcode_helpline']);
 
-		$custom_tags = array(
-			'BBCODE_NAME'		=> "'[{$row['bbcode_tag']}]', '[/" . str_replace('=', '', $row['bbcode_tag']) . "]'",
+		// Does the closing bbcode tag exists? If so display it.
+		$bbcode_close_tag	= '%\[\/' . utf8_strtolower($row['bbcode_tag']) . ']%';
```

Comment:
doesn't `]` need to be escaped?

Reply:
IMHO is not necessary.
Test; https://regex101.com/r/NxgOZv/1

Reply:
I would still opt for escaping it to have it uniform.